spec/ddtrace/transport/http/adapters/net_integration_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+require 'stringio'
+require 'thread'
+require 'webrick'
+
+require 'ddtrace/transport/http'
+require 'ddtrace/transport/http/adapters/net'
+
+RSpec.describe 'Adapters::Net integration tests' do
```

Comment:
Were these missing before or transplanted from removed spec files?

Reply:
They never existed: we had similar specs in `meta_header_spec.rb` and `http_integration_spec.rb` but none that explicitly tested `Net::HTTP` adapter in such a way.